### PR TITLE
Feature/cypress user login flow

### PR DESCRIPTION
**Pull request by nataliemcintyre2021**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- Successfully intercept and stub existing user log-in flow

## What (if anything) did you refactor?
- Renamed SignIn Component to ExistingLogIn

## Were there any issues that arose?
- Moving forward on tests - currently researching creating test for log in of new user

## Is there anything that you need from your teammates?
-

## Any other comments, questions, or concerns?
-

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor

## How Has This Been Tested?
- [ ] open index.html
- [ ] console.log()
- [ ] dev tools
- [X] Cypress

## Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: cypress/integration/existing_log_in_spec.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-describe('Signin', () => {
+describe('Existing Log In', () => {
```
Comment (hayleyw7) [inline cypress/integration/existing_log_in_spec.js:1]: Our rename helps so much. Glad we did this.

File: cypress/integration/profile_spec.js [hunk 2]
```diff
@@ -4,46 +4,41 @@ describe('Profile', () => {
...
-
-  // it('should logout', () => {
-  //   cy.get('button[class="nav-btn logout-btn"]')
-  //     .click()
-  //   .get('button[class="create-btn"]')
-  //     .should('be.visible')
-  // })
-})
\ No newline at end of file
+  it('should render header', () => {
```
Comment (hayleyw7) [inline cypress/integration/profile_spec.js:7]: All passing!

File: cypress/integration/user_display_spec.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-  // these won't work until we can actually stub & create new accounts each test
-
-  it('should render header', () => {
-    cy.get('h1')
-      .contains('EnneaLink')
-  })
+// describe('User Display', () => {
+//
+//   beforeEach(() => {
+//     cy.loadUserDisplayPage()
```
Comment (hayleyw7) [inline cypress/integration/user_display_spec.js:4]: Glad we caught this.

File: cypress/support/commands.js [hunk 4]
```diff
@@ -23,32 +23,113 @@ Cypress.Commands.add('loadEditProfilePage', () => {
...
-//   cy.intercept(`${url}`, {
-//     fixture: `${fixturePage}_test_data.json`,
-//   })
+// Cypress.Commands.add('loadUserDisplayPage', () => {
+//   cy.loadProfilePage()
+//     .get('button[class="nav-btn friends-btn"]')
+//       .click()
+//       .interceptUserStats()
+//       .wait(1000)
+//       .interceptGetAllUsers()
```
Comment (hayleyw7) [inline cypress/support/commands.js:40]: Not working yet, but we're going to try to resolve this if we can.

File: cypress/support/commands.js [hunk 5]
```diff
@@ -23,32 +23,113 @@ Cypress.Commands.add('loadEditProfilePage', () => {
...
 // })
 
-// stubbing endpoints
-
-// Cypress.Commands.add('loadProfile', (g) => {
-//   cy.loadApp()
-//     .interceptAPI('profile', 'http://localhost:4500/graphql')
-// })
+Cypress.Commands.add('interceptUserStats', () => {
```
Comment (hayleyw7) [inline cypress/support/commands.js:43]: This is so good! I'm so proud of us for figuring out how to stub with GraphQL.

File: cypress/support/commands.js [hunk 6]
```diff
@@ -23,32 +23,113 @@ Cypress.Commands.add('loadEditProfilePage', () => {
...
+              }
+            }
+          }
+        })
+      }
+    }
+  )
+})
 
+Cypress.Commands.add('interceptGetAllUsers', () => {
```
Comment (hayleyw7) [inline cypress/support/commands.js:80]: not working yet but great progress

File: src/components/ExistingLogIn/ExistingLogIn.js [hunk 7]
```diff
@@ -1,10 +1,10 @@
-import './SignIn.css';
+import './ExistingLogIn.css';
```
Comment (hayleyw7) [inline src/components/ExistingLogIn/ExistingLogIn.js:1]: much less confusing name

**Review by hayleyw7 — APPROVED** [review]
We got super great progresso on this. Go team!

**Review by MTCawthray — APPROVED** [review]
Great user flow testing. So happy we got one intercept working so far!

**Pull request merged**